Just adding more to our docs section

cupokyle opened the pull request:
flesh out docs folder a bit more with reference to final project setup on compass

zeeplo reviewed (commented):
Maybe we could add some of the markdown files to README for easier reading and access

cupokyle commented:
> Maybe we could add some of the markdown files to README for easier reading and access
Just removed some of the notes files and updated the README with the information instead!

zeeplo commented:
update readme
